src/status_im/handlers.cljs

```diff
@@ -42,6 +45,18 @@
   (fn [db [_ k v]]
     (assoc-in db [:animations k] v)))
 
+(register-handler :open-sharing
+                  (u/side-effect!
```

Comment:
The spacing here is too big. I think it is better to make it look the way it looked before.